os/arch/boardctl.c

```diff
@@ -150,7 +151,7 @@ int boardctl(unsigned int cmd, uintptr_t arg)
 		sched_lock();
 		/* Add 100ms delay for flushing another logs like printf. */
 		up_mdelay(100);
-		lldbg("Board will Reboot now.\n");
+		lldbg("Board will Reboot now. pid: %d\n", getpid());
```

Comment:
Let's add some sleep before the board_sleep. I found many logs which do not print entire message.